dvc/remote/ssh/__init__.py

```diff
@@ -63,6 +72,31 @@ def __init__(self, repo, config):
             "port": self.port,
         }
 
+    @staticmethod
+    def _load_user_ssh_config(hostname):
+        user_config_file = os.path.expanduser("~/.ssh/config")
```

Comment:
https://github.com/iterative/dvc.org/pull/290#discussion_r281037621 Let's use `os.path.join("~", ".ssh", "config")` instead, since on windows it won't resolve it properly.

Reply:
accepted, will fix it after test passed on Travis